### PR TITLE
Add trace-cmd

### PR DESCRIPTION
**Pull request by aadhar-agarwal**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [X] The toolchain has been rebuilt successfully (or no changes were made to it)
- [X] The toolchain/worker package manifests are up-to-date
- [X] Any updated packages successfully build (or no packages were changed)
- [X] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [X] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [X] All package sources are available
- [X] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [X] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [X] All source files have up-to-date hashes in the `*.signatures.json` files
- [X] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [X] Documentation has been updated to match any changes to the build system
- [X] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
- This PR adds trace-cmd utility into 3.0

###### Change Log  

- Add trace-cmd and its dependencies libtracecmd and libtracefs

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- https://microsoft.visualstudio.com/OS/_workitems/edit/48554342
- https://msazure.visualstudio.com/One/_workitems/edit/26516788

###### Test Methodology

- Buddy build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=551505&view=results
- Full build: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=551508&view=results

- Used the trace-cmd binary in a 3.0 Azure VM through the following:
image

### CODE REVIEW

File: SPECS/libtracefs/libtracefs.spec [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+%package devel
+Summary: Development headers of %{name}
+Requires: %{name}%{_isa} = %{version}-%{release}
+
+%description devel
+Development headers of %{name}
+
+%prep
+%setup -q
```
Comment (mfrw) [inline SPECS/libtracefs/libtracefs.spec:42]: NIT
```suggestion
%autosetup -p1
```

File: SPECS/trace-cmd/trace-cmd.spec [hunk 2]
```diff
@@ -0,0 +1,108 @@
+# git tag
+#%%global git_commit trace-cmd-v2.6.2
+#%%global git_commit 57371aaa2f469d0ba15fd85276deca7bfdd7ce36
+
+%global srcversion 3.2
+
+Name:          trace-cmd
+Version:       %{srcversion}
```
Comment (mfrw) [inline SPECS/trace-cmd/trace-cmd.spec:8]: I recall that the auto-upgarde pipeline used to have issues with a macro in the version.
I hope that is not the case now.
/cc @Nicogbg
```suggestion
Version:       %{srcversion}
```

**Review by mfrw — APPROVED** [review]

File: SPECS/libtracefs/libtracefs.spec (hunk 1 above)
Comment (aadhar-agarwal) [reply SPECS/libtracefs/libtracefs.spec]: Updated!

File: SPECS/trace-cmd/trace-cmd.spec (hunk 2 above)
Comment (aadhar-agarwal) [reply SPECS/trace-cmd/trace-cmd.spec]: Synced with Nicolas offline, will not use a macro in the version. Updated trace-cmd.spec file

**Review by mfrw — APPROVED** [review]
LGTM

**Pull request merged**